runtime/syntax/man.vim

```diff
@@ -5,18 +5,21 @@ if exists('b:current_syntax')
   finish
 endif
 
-syntax case  ignore
-syntax match manReference      display '[^()[:space:]]\+([0-9nx][a-z]*)'
-syntax match manSectionHeading display '^\S.*$'
-syntax match manTitle          display '^\%1l.*$'
-syntax match manSubHeading     display '^ \{3\}\S.*$'
-syntax match manOptionDesc     display '^\s\+\%(+\|-\)\S\+'
+" Not contained because sometimes, manpages contain garbage backspaced
+" characters. See Restricted Shell at bash(1)
```

Comment:
You mentioned it in the original rewrite PR, but I thought it only applied for characters that repeated themselves after the backspace. so `x\bx` where x was a character because this is the regex you used on the PR you made on my fork. But nice catch, I've fixed it now for this PR, those should be highlighted correctly now.